webapp/src/constants.js

```diff
@@ -0,0 +1,7 @@
+export const MessageTypes = ['BSM', 'SSM', 'SPAT', 'SRM', 'MAP']
+export const DotName = 'CDOT'
```

Comment:
We'll probably want to pull at least the DotName and mapbox viewstate values out to environment vars here to make it easily configurable

Reply:
I reworked these variables to all be environmental variables.